scripts/run_tracker.py

```diff
@@ -87,7 +87,7 @@ def is_transferred(run, transfer_file):
                 #Rows have two columns: run and transfer date
                 if row[0] == os.path.basename(run):
                     return True
-        if os.path.exists(os.path.join(run, 'transferring')):
+        if os.path.exists(os.path.join(run, '.transferring')):
```

Comment:
Why name a checkpoint file with a dot (.). Isn't that just confusing for others if they want to inspect / debug a transfer in progress?